### PR TITLE
Intermediate Results 2

### PR DESCRIPTION
**Pull request by joshday**
Okay, I think I've got this working for `simulate` with the following information being shared via RabbitMQ:

- `job_id`: The job id.
- `iter`: Current iteration.
- `time`: Current time.
- `params`: Current parameter estimates.
- `retcode`: (https://docs.sciml.ai/DiffEqDocs/stable/basics/solution/#retcodes) could possibly provide some useful information back to the user if things go wrong.

### CODE REVIEW

**Comment by joshday** [comment]
Ref: https://github.com/DARPA-ASKEM/simulation-service/issues/34

TODOs:

- calibrate
- ensemble (when it gets added)

File: src/operations/Operations.jl [hunk 1]
```diff
@@ -43,13 +110,13 @@ function calibrate(; model, dataset::DataFrame, context)
...
-NOT IMPLEMENTED
-"""
-function ensemble(; models::AbstractArray{AbstractPetriNet}, timespan=(0.0, 100.0)::Tuple{Float64, Float64})
+#-----------------------------------------------------------------------------# ensemble
+function ensemble(; models::AbstractArray{AbstractPetriNet}, timespan::Tuple{Float64, Float64} = (0.0, 100.0), context=nothing)
     throw("ENSEMBLE IS NOT YET IMPLEMENTED")
 end
 
+
+# joshday: what is this function?
```
Comment (ChrisRackauckas) [inline src/operations/Operations.jl:119]: it looks like it's just `calibrate` but with a global optimization done instead. It should instead just be an option in `calibrate` set by the context.

File: src/service/ArgIO.jl [hunk 2]
```diff
@@ -15,11 +15,11 @@ include("./AssetManager.jl"); import .AssetManager: fetch_dataset, fetch_model,
 export prepare_input, prepare_output
 
 
-"""
-Transform requests into arguments to be used by operation    
+# """
```
Comment (ChrisRackauckas) [inline src/service/ArgIO.jl:18]: how come the docstrings are all commented out?

**Review by fivegrant — APPROVED** [review]

File: src/service/ArgIO.jl (hunk 2 above)
Comment (joshday) [reply src/service/ArgIO.jl:18]: There were warnings about replaced docstrings when loading the package.

**Pull request merged**